subprojects/composite-builds/src/main/java/org/gradle/composite/internal/DefaultIncludedBuildController.java

```diff
@@ -85,6 +85,11 @@ public void startTaskExecution() {
         started.countDown();
     }
 
+    @Override
+    public void stopTaskExecution() {
```

Comment:
Should we remove `implements Stoppable`? Seems like we will now be stopping this twice.